src/Analysis/Core/Impl/DependencyResolution/PathResolverSnapshot.cs

```diff
@@ -81,28 +81,26 @@ public PathResolverSnapshot(PythonLanguageVersion pythonLanguageVersion, string
             Version = version;
         }
 
-        public ImmutableArray<string> GetAllModuleNames() => GetModuleNames(_roots.Prepend(_nonRooted));
-        public IEnumerable<string> GetInterpreterModuleNames() => GetModuleNames(_roots.Skip(_userRootsCount).Append(_builtins));
-
-        private ImmutableArray<string> GetModuleNames(IEnumerable<Node> roots) {
+        public ImmutableArray<string> GetAllImportableModuleNames(bool includeImplicitPackages = true) {
```

Comment:
Do you use this parameter in production?

Reply:
Doesn't seem that way, is this what should be false if the interpreter is 2.x?

Reply:
Nope

Reply:
I can just use `LanguageVersionRequiresInitPy` instead.

Reply:
Ah yes, that's better.